lib/Utils/Paths.cpp

```diff
@@ -0,0 +1,229 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author:  Jermaine Dupris <support@greyangle.com>
```

Comment:
I'd prefer, for now, to remove the authorship. I will update the CREDITS file and I will mention him in the next release notes.